rex/utilities/bc_utils.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+rex bias correction utilities.
+"""
+import json
+import numpy as np
+import logging
+from warnings import warn
+import rex.bias_correction
+
+
+logger = logging.getLogger(__name__)
+
+
+def _parse_bc_table(bc_df, gids):
+    """Parse the bias correction table for required bc functions and kwargs
+
+    Parameters
+    ----------
+    bc_df : pd.DataFrame
+        DataFrame with wind or solar resource bias correction table. This
+        must have columns "gid" and "method", where "gid" is the resource
+        file indices, and "method" is a function name from the
+        ``rex.bias_correction`` module. Only windspeed or GHI+DNI+DHI are
+        corrected, depending on the technology. See the
+        ``rex.bias_correction`` module for more details on available
+        bias correction methods.
+    gids : list | np.ndarray
+        Array of integer gids (spatial indices) from the source h5 file.
+        This is used to get the correct bias correction parameters from
+        ``bias_correct`` table based on its ``gid`` column
+
+    Returns
+    -------
+    bc_fun : function
+        Function from ``rex.bias_correction`` to use.
+    bc_fun_kwargs : dict
+        Kwargs from ``bc_df`` to input to ``bc_fun``. This may include extra
+        kwargs that are not required by ``bc_fun`` and should be cleaned before
+        passing to the function.
+    bool_bc : np.ndarray
+        1D Boolean array with length equal to the ``gids`` input with ``True``
+        where data has available bias correction inputs in ``bc_df`` and
+        ``False`` where not
+    """
+
+    if 'method' not in bc_df:
+        msg = ('Bias correction table provided, but "method" column not '
+               'found! Only see columns: {}. Need to specify "method" which '
+               'is a function name from `rex.bias_correction`'
+               .format(list(bc_df.columns)))
```

Comment:
Did you want to add a `logger.error(msg)` call here like for the exception below?

Reply:
missed this, thanks!